### DISCUSSION TITLE
[UI/UX] profile name editable

### DISCUSSION THREAD

**Discussion by dev0pa** [iotaledger/firefly#1573] [Features / Enhancements]
make profile name editable

**Comment by maxwellmattryan** [comment]
Hey @dev0pa, closing this issue and have converted into a discussion 👍

**Comment by kajover** [comment]
@maxwellmattryan link to discussion is not working or discussion was removed.

**Reply by maxwellmattryan** [reply]
My bad! This is the discussion I was mentioning in the other thread.

**Comment by kajover** [comment]
Version 1.2.0 - Renaming is not possible. Are there any updates on that feature?
A good placement may be under "All Settings" and "Security" (although it does not really have anything to do with security, see screenshot.
<img width="652" alt="Screen Shot 2021-12-07 at 15 22 29" src="https://user-images.githubusercontent.com/83403952/145046585-17f2566a-e0b9-4d82-951a-39c4c9653c95.png">

**Reply by maxwellmattryan** [reply]
Perhaps in the future we can add more profile settings and separate the menu in five settings like "General", "Profile", "Security", "Advanced", and "Help". I have seen others ask for stuff like customized profile avatars too ~

**Reply by maxwellmattryan** [reply]
I think another good enhancement for Ledger profiles specifically would be the ability to lock an account or your profile from view or access unless the IOTA Ledger app is opened on the device itself.
